### PR TITLE
New in chrome 106 post

### PR DESCRIPTION
**Pull request by tropicadri**
Exciting news for 106!

### CODE REVIEW

**Review by petele — APPROVED** [review]

File: site/en/blog/new-in-chrome-106/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: New in Chrome 106
+description: >
+  Chrome 106 is rolling out now. There are new Intl APIs to give you more control when formatting numbers. There’s an origin trial for the new Pop Up API, making it easy to surface critical content to the user. There are a handful of CSS improvements. And there’s plenty more
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-106/index.md:4]:
```suggestion
  Chrome 106 is rolling out now. There are new Intl APIs to give you more control when formatting numbers. There’s an origin trial for the new Pop Up API, making it easy to surface critical content to the user. There are a handful of CSS improvements. And there’s plenty more.
```

File: site/en/blog/new-in-chrome-106/index.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+{% Codepen {
+    user: 'tropicadri',
+    id: 'OJZmgOg',
+    height: 450,
+    tab: 'result'
+  }
+%}
+
+CSS Grid support for interpolation for `grid-template-columns` and `grid-template-rows` is coming, it was planned for 106, but is delayed and will be launched in Chrome 107, you can still try it out in Chrome Beta. Here is Bramu’s code as an example:
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-106/index.md:173]:
```suggestion
CSS Grid support for interpolation for `grid-template-columns` and `grid-template-rows` is coming, it was planned for 106, but is delayed and will be launched in Chrome 107, you can still try it out in Chrome Beta. Here is Bramus' code as an example:
```

File: site/en/blog/new-in-chrome-106/index.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+{% Codepen {
+    user: 'bramus',
+    id: 'JjvEExW',
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-106/index.md:177]:
```suggestion
    user: 'web-dot-dev',
    id: 'XWqVowx'
```

File: site/en/blog/new-in-chrome-106/index.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+It is a unit designed to size text, allowing you to limit width to improve readability, and it gives predictable control regardless of the text size.
+
+For example if you set the `max-width` of a container, let's say to 10ic,
+you *know* that container will contain at most 10 full width glyphs,
+no matter the font-size. Check it out in the following example:
+
+{% Codepen {
+    user: 'tropicadri',
+    id: 'OJZmgOg',
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-106/index.md:167]:
```suggestion
    user: 'web-dot-dev',
    id: 'eYrybaj',
```

**Review by rachelandrew — APPROVED** [review]
Couple of nits plus forking the CodePens to the team account. Otherwise LGTM.

**Review by tropicadri — COMMENTED** [review]
fixing typo

File: site/en/blog/new-in-chrome-106/index.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+    height: 450,
+    tab: 'result'
+  }
+%}
+
+CSS Grid support for interpolation for `grid-template-columns` and `grid-template-rows` is coming, it was planned for 106, but is delayed and will be launched in Chrome 107, you can still try it out in Chrome Beta. Here is Bramus' code as an example:
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'XWqVowx'
```
Comment (tropicadri) [inline site/en/blog/new-in-chrome-106/index.md:177]:
```suggestion
    id: 'XWqVowx',
```

**Pull request merged**